src/Oracle/Helper.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Text;
+
+namespace Neo.Plugins
+{
+    internal static class Helper
+    {
+        public static string ReadUserInput(string prompt, bool password = false)
```

Comment:
Maybe we should be able to use the `ConsoleService` and reuse this method

Reply:
This can be done by neo-project/neo#2068

Reply:
Please check neo-project/neo-node#679

Reply:
Done